File: GrammerBasics/src/Article.java

public class Article extends Grammer {

String art;

public Article(String art) {
	super();
	this.art = art;
}

public String getArt() {
	return art;
}

public void setArt(String art) {
	this.art = art;
}

@Override
public String printSelf() {
	
	return art;
}
@Override
public String toString() {
	return art;
}
}

File: JabberJaw/Grammer/Article.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Grammer
{
    class Article : Grammer
    {

        String art;

public Article(String art)
    {
        this.art = art;
    }

    public String getArt()
    {
        return art;
    }

    public void setArt(String art)
    {
        this.art = art;
    }
        public override string printSelf()
        {
            return art;
        }
    }
}
